python/lsst/afw/math/background.py

```diff
@@ -1,223 +1,12 @@
 from __future__ import absolute_import
-from builtins import object
-#
-# LSST Data Management System
-# Copyright 2008, 2009, 2010 LSST Corporation.
-#
-# This product includes software developed by the
-# LSST Project (http://www.lsst.org/).
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the LSST License Statement and
-# the GNU General Public License along with this program.  If not,
-# see <http://www.lsstcorp.org/LegalNotices/>.
-#
-import os
-import lsst.daf.base as dafBase
-from lsst.log import Log
-import lsst.afw.geom as afwGeom
-import lsst.afw.image as afwImage
-from lsst.afw.fits import FitsError, MemFileManager, reduceToFits
-from . import mathLib as afwMath
 
-class BackgroundList(object):
-    """A list-like class to contain a list of (afwMath.Background, interpStyle, undersampleStyle) tuples
+from ._background import *
```

Comment:
Please don't use `from foo import *`. In this case you need only import `Background`.

Also, I hate to say this, but that licensing boilerplate probably ought to be restored. But suit yourself.